test/helpers.py

```diff
@@ -17,8 +17,11 @@
 # IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
 # CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
+import contextlib
+import gc
 import equinox as eqx
 import typeguard
+from typing import Any, Callable
```

Comment:
Nit: `typing.Callable` is deprecated; prefer `from collections.abc import Callable`.